src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+# Metrics
+
+This section describes how to enable metrics reporting on the Policy Server.
+
+We are going to use [Prometheus](https://prometheus.io/) to scrape metrics exposed by the Policy
+Server.
+
+## Install Prometheus
+
+We will use the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator),
+that allows us to intuitively define Prometheus' Targets.
+
+There are many ways to install and set up prometheus. For ease of deployment, we will use the
+Prometheus community helm chart.
+
+Let's add the helm repository from the Prometheus Community:
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+```
+
+Now, let's install the
+[`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+chart. This chart contains a collection of Kubernetes manifests, Grafana dashboards, and Prometheus
+rules.
+
+Let's create a `kube-prometheus-stack-values.yaml` file to configure the `kube-prometheus-stack`
+Helm chart values with the following contents:
+
+```yaml
+prometheus:
+  additionalServiceMonitors:
+    - name: kubewarden
+      selector:
+        matchLabels:
+          app: kubewarden-policy-server-default
+      namespaceSelector:
+        matchNames:
+          - kubewarden
+      endpoints:
+        - port: metrics
+          interval: 10s
+```
+
+The `prometheus-operator` deployed as part of this Helm chart defines the concept of [Service
+Monitors](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/design.md#servicemonitor),
+used to declaratively define what services should be monitored by Prometheus.
+
+In our case, we are adding a Service monitor targeting the `kubewarden` namespace, for services that
+match labels `app=kubewarden-policy-server-ha-policy-server`. This way, the Prometheus Operator is
```

Comment:
The label provided here is different from the one specified in the yaml example from above